app/main.py

```diff
@@ -2,0 +1,17 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    mask_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (VaccineError):
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            mask_to_buy += 1
```

Comment:
The logic in this part of the code is a bit unclear. If one of the friends is not vaccinated, the function immediately returns a message that all friends should be vaccinated. However, if one of the friends is not wearing a mask, the function continues to check the other friends. This inconsistency may lead to unexpected behavior. If the function should stop checking as soon as it finds a friend who is not vaccinated or not wearing a mask, you can raise an exception as soon as a friend without a mask is found, similar to how you handle the VaccineError.